src/tree/containers/ContainerProperties.ts

```diff
@@ -65,6 +66,8 @@ export function getContainerPropertyValue(item: ListContainersItem, property: Co
             return getComposeProjectName(item);
         case 'Image':
             return item.image.originalName;
+        case 'Label':
+            return getLabel(item?.labels, 'containers');
```

Comment:
A couple of changes I'd suggest to this method--

1. Change the name from `getLabel` to `getConfiguredLabelGroup`, since it's not taking the label to get as an input, but rather looking it up from settings
2. Pass in just `item` and in the method body, go into `item.labels`

Reply:
That makes sense. Makes the code more readable for sure.